src/submap.cpp

```diff
@@ -160,7 +160,7 @@ void submap::update_legacy_computer()
     if( legacy_computer ) {
         for( int x = 0; x < SEEX; ++x ) {
             for( int y = 0; y < SEEY; ++y ) {
-                if( ter[x][y] == t_console ) {
+                if( frn[x][y] == furn_str_id( "f_console" ) ) {
```

Comment:
If it's a legacy computer, then the save may still contain "t_console" which should also be updated, unless we actually migrate "t_console" to "f_console" while loading the save.